<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FC_FI_batelada</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,3,1">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>FI_config</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Pot. Off</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="potOff_SB">
         <property name="maximum">
          <double>100000000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>FC_config</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="max_RB">
           <property name="text">
            <string>Pot. Max</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="potNom_RB">
           <property name="text">
            <string>Pot. Nominal</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="potProc_RB">
           <property name="text">
            <string>Pot. Proc.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
         <item>
          <widget class="QRadioButton" name="quantil_RB">
           <property name="text">
            <string>Quantil</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="quantil_SB"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aplicar_PB">
      <property name="text">
       <string>Aplicar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
